oh-my-git.plugin.zsh

```diff
@@ -224,9 +225,9 @@ function oh_my_git_info {
 				fi
 		
 				oh_my_git_string+="
-					(${current_branch_color}${current_branch}${reset}
+					${branch_color}(${current_branch_color}${current_branch}${reset}
```

Comment:
I'm not sure how how `branch_color` is useful here since it will be overridden by `current_branch_color` anyway, since when on a branch this will always be the current branch.

However this would be useful for the upstream branch color, but it should be renamed then to `upstream_branch_color`.

Reply:
I use your plugin in a customized prompt. I had problems choosing the text colors because they were reset over and over again.

The new variable ('branch_color') is used to set the general text color of the branch segment. I always want the text in this segment to be black. 

without:
![without_branch_color](https://user-images.githubusercontent.com/4189991/37621315-db7bb2b8-2bbe-11e8-9953-d57d35d5c711.png)

with:
![with_branch_color](https://user-images.githubusercontent.com/4189991/37621208-8f355a08-2bbe-11e8-80ed-0e43c017a761.png)